src/pages/computer-hardware/cpu/index.md

```diff
@@ -1,15 +1,20 @@
 ---
 title: Cpu
```

Comment:
Can you capitalise CPU here please, this is how the article is displayed in the menu